Code/GraphMol/MolDraw2D/test1.cpp

```diff
@@ -825,6 +831,26 @@ void testGithub774() {
   std::cerr << " Done" << std::endl;
 }
 
+void test9MolLegends() {
+  std::cout << " ----------------- Test 9 (molecule legends)" << std::endl;
+  {
+    std::string smiles = "CC[13CH2][CH2:7][CH-]C[15NH2+]C";
+    std::string nameBase = "test5_1";
+    ROMol *m = SmilesToMol(smiles);
+    TEST_ASSERT(m);
+    RDDepict::compute2DCoords(*m);
+    WedgeMolBonds(*m, &(m->getConformer()));
+    MolDraw2DSVG drawer(300, 300);
+    drawer.drawMolecule(*m, "mol legend");
+    drawer.finishDrawing();
+    std::string txt = drawer.getDrawingText();
+    std::ofstream outs("test9_1.svg");
+    outs << txt;
+    // TEST_ASSERT(txt.find("<svg:svg")!=std::string::npos);
```

Comment:
including an actual test isn't a bad idea. 
The big thing with most of these "tests" is (at the moment) really to generate the output files so that a human can look at them. @bp-kelley and I have talked about doing some real testing for at least things like stereochemistry, but that's mainly an interesting intellectual exercise.